app/db/seeds/exercise-records-seed-generator.ts

```diff
@@ -0,0 +1,44 @@
+import { 
+  ExerciseRecordSnapshotIn, 
+  Workout,
+  WorkoutSet,
+  WorkoutSetSnapshotIn, 
+} from "app/db/models"
+import { getSnapshot } from "mobx-state-tree";
+
+export const getRecords = (workouts: Workout[]): ExerciseRecordSnapshotIn[] => {
+  const records: ExerciseRecordSnapshotIn[] = [];
```

Comment:
Wouldn't an object structure be easier to use down the line than an array?
An array would mean you have to filter out records by exercise every time.